request-manager/grapher/spec.go

```diff
@@ -91,6 +91,12 @@ func ReadConfig(configFile string) (Config, error) {
 
 	for sequenceName, sequence := range cfg.Sequences {
 		sequence.Name = sequenceName
+		for _, arg := range sequence.Args.Static {
```

Comment:
Actually, could you do the same thing for Optional args? They should also always have a default.